docker/test_generate_makefile.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for generate_makefile.py."""
+
+from unittest.mock import patch, call
```

Comment:
We don't do multiple imports on a single line https://google.github.io/styleguide/pyguide.html#313-imports-formatting
can you check the styleguide for other violations please?

Reply:
Done